oauth2_http/java/com/google/auth/oauth2/IdToken.java

```diff
@@ -77,6 +78,7 @@ public static IdToken create(String tokenValue) throws IOException {
    * @return returns com.google.auth.oauth2.IdToken
    */
   public static IdToken create(String tokenValue, JsonFactory jsonFactory) throws IOException {
+    tokenValue = tokenValue.trim();
```

Comment:
add a test to catch regressions?

Reply:
just adding test to validate that 0x20 is handled by java lib if it is part of the token in an expected way